tracer/src/Datadog.Trace/Agent/Api.cs

```diff
@@ -140,15 +183,72 @@ public async Task<bool> SendTracesAsync(ArraySegment<byte> traces, int numberOfT
                     continue;
                 }
 
-                _log.Debug<int>("Successfully sent {Count} traces to the DD agent", numberOfTraces);
                 return true;
             }
         }
 
-        private async Task<bool> SendTracesAsync(ArraySegment<byte> traces, int numberOfTraces, IApiRequest request, bool finalTry)
+        private async Task<bool> SendStatsAsyncImpl(IApiRequest request, bool isFinalTry, SendStatsState state)
+        {
+            bool success = false;
+
+            // Set additional headers
+            if (_containerId != null)
+            {
+                request.AddHeader(AgentHttpHeaderNames.ContainerId, _containerId);
+            }
+
+            var stream = new MemoryStream();
```

Comment:
Dispose is useless on MemoryStream. But I guess I can add it for cleanliness.

Reply:
It didn't event occured to me it would be useless even though it seems quite logical :) TBH, I see `Stream`, I think `Dispose`. But now that I understand, I don't mind (even though it doesn't heart to have it as well :p)